andela/blackninjas-frontend
# #171974960 Users should be able to search in their trip requests

#### What does this PR do?
this PR comes to add the title to the main layout
#### Description of Task to be completed?

 **Why**
As I should  see the title of the page on the top



#### How should this be manually tested?
After clone repository goes to the root of the project open path into command run the npm install and once setup is done type npm install.

after that type npm start you will get a sign-in page then sign in and you will be redirected to
the dashboard after successfully sign in
and then you will see the title on the top of the page which indicates the current page you are on.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#171974960](https://www.pivotaltracker.com/story/show/171974960)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/47531860/77756450-de1a5000-7037-11ea-8edf-95ffc7c2eb4e.png)

![image](https://user-images.githubusercontent.com/47531860/77756568-19b51a00-7038-11ea-9cd2-83e8e522ebc0.png)


#### Questions:
N/A
